pkgs/top-level/python-packages.nix

```diff
@@ -10366,7 +10366,7 @@ in {
 
   spacy-loggers = callPackage ../development/python-modules/spacy-loggers { };
 
-  spacy_models = callPackage ../development/python-modules/spacy/models.nix { };
+  spacy_models = callPackage ../development/python-modules/spacy/models.nix { inherit (pkgs) jq; };
```

Comment:
Weirdly, when calling the updateScript of spacy_models `jq` got interpreted as `python3Packages.jq`. This is the best fix I could think of.